ddtrace/contrib/django/apps.py

```diff
@@ -1,13 +1,14 @@
 import logging
 
 # 3rd party
-from django.apps import AppConfig
+from django.apps import AppConfig, apps
 
 # project
 from .db import patch_db
 from .conf import settings
 from .cache import patch_cache
 from .templates import patch_template
+from .rest_framework import patch_rest_framework
```

Comment:
Not sure we want to import it all the time, because it would trigger this exception log https://github.com/DataDog/dd-trace-py/pull/389/files#diff-6f7dbfbaaae5d3b33ba40f455fe6c12fR7

We should probably test the existence or usage of REST framework before trying to import it.

Reply:
Totally agree!